docs/src/dev-guide/tooling-containers.md

```
@@ -35,6 +35,33 @@ distros using glibc 2.28+, including:
+  components/core/tools/docker-images/clp-env-base-musllinux_1_2-aarch64
+  ```
+
+### clp-core-dependencies-x86-musllinux_1_2
+
+* Path:
+
+  ```text
+  components/core/tools/docker-images/clp-env-base-musllinux_1_2-x86_64
+  ```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)